twake/frontend/src/app/components/Leftbar/Channel/Channel.scss

```diff
@@ -378,3 +378,12 @@
     }
   }
 }
+
+.ant-avatar-group .ant-avatar:not(:first-child) {
```

Comment:
Since you are overriding the antd classnames, you coule define this in the `theme.less` file.

Reply:
only if this is something we want to be global to all twake (just in case :) )

Reply:
It's only here where we'll have a group of avatars so it's quite the same.